mypy/checker.py

```diff
@@ -6285,6 +6285,7 @@ def builtin_item_type(tp: Type) -> Optional[Type]:
             "builtins.dict",
             "builtins.set",
             "builtins.frozenset",
+            "_collections_abc.dict_keys",
```

Comment:
What about `typing.KeysView`? https://github.com/python/cpython/blob/70fc9641b56144854777aef29c145cd10789e3df/Lib/typing.py#L2701

And `collections.abc.KeysView`?
What about `ValuesView` and `ItemsView`?

Reply:
And `dict_values` as well as `dict_items` 🙂

Reply:
> What about typing.KeysView? 

Sounds good. I wasn't sure exactly what would be best to be included in this changeset. Let me drag all of those in too

Reply:
> What about `typing.KeysView`? https://github.com/python/cpython/blob/70fc9641b56144854777aef29c145cd10789e3df/Lib/typing.py#L2701

This is included with 75ca813a0428a8db2be7c356535e123fb5da4bac